package.json

```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "fs-extra": "^0.30.0",
     "memorystream": "^0.3.1",
-    "require-from-string": "^1.1.0",
+    "require-from-string": "1.2.0",
```

Comment:
Is there a good reason for fixing it to `1.2.0`? Can we just go with `^1.2.0`?

Reply:
Unsure - ended up using a constructor, so didn't need browser solc and the change in this PR. False alarm.

I think ^1.2.0 is ok, but I haven't tested it.